kernel/process/process.rs

```diff
@@ -390,7 +390,10 @@ impl Process {
         let current = current_process();
         if let Some((signal, sigaction)) = current.signals.lock().pop_pending() {
             let sigset = current.sigset.lock();
-            if !sigset.get(signal as usize).unwrap_or(true) {
+            if signal == SIGSTOP
+                || signal == SIGKILL
```

Comment:
I think we don't need these checks. When we implement them in the future, we won't respect `sigaction` and will force killing (`SIGKILL`) or pausing (`SIGSTOP`) the target process.

Reply:
Removed